### DISCUSSION TITLE
linked folders synchronization issue

### DISCUSSION THREAD

**Discussion by liberamiguel** [abraunegg/onedrive#1883] [Q&A] [answered]
Hello,

I'm experiencing issues with synchronization of linked folders (folder shared with me in sharepoint/teams) which synchronizes correctly when using official windows OneDrive client

With this client I can't even see the linked folders on the list or trying to synchronize

Has anyone had similar issue?

**Comment by abraunegg** [comment, marked as answer]
@liberamiguel 

Linked folders - those that are added to your OneDrive via the 'Add Shortcut to My Files' is not supported by the OneDrive API - see https://github.com/abraunegg/onedrive/issues/1224

To configure SharePoint libraries you must follow https://github.com/abraunegg/onedrive/blob/master/docs/SharePoint-Shared-Libraries.md
